content/docs/explanation/netzgrafik-editor/_index.en.md

```diff
@@ -7,75 +7,35 @@ description: "Open-source software developed by SBB CFF FFS and its integration
 
 Netzgrafik-Editor (NGE) is an open-source software that enables the creation, modification, and analysis of regular-interval timetable, at a macroscopic level of detail, developed by [Swiss Federal Railways (SBB CFF FFS)](https://www.sbb.ch/). See [front-end](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend) and [back-end](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-backend) repositories.
 
-OSRD (microscopic level of detail, trains scheduled once, based on a defined infrastructure, depicts a timetable) and NGE (macroscopic level of detail, regular-interval based train runs, no infrastructure, depicts a transportation plan) are semantically different, but close enough to make it work together.
+OSRD (microscopic level of detail, based on a in-quality defined infrastructure, depicting a timetable composed of unique train schedules) and NGE (macroscopic level of detail, based on no explicit infrastructure, depicting a transportation plan composed on regular-interval based train runs) are semantically different, but close enough to make it work together.
+
 The compatibility between NGE and OSRD has been tested through a proof of concept, by running both pieces of software as separate services and without automated synchronization.
 
-The idea is to give to OSRD a graphical tool to edit (create, update and delete train schedules from) a timetable from an operational study scenario, and get some insights on analytics at the same time. The second benefit of using both microscopic and macroscopic level of detail is that OSRD microscopic calculations can be propagated in NGE to enhance the level of detail of it.
+The idea is to give to OSRD a graphical tool to edit (create, update and delete train schedules from) a timetable from an operational study scenario, and get some insights on analytics at the same time. The second benefit of using both microscopic and macroscopic level of detail is that OSRD's microscopic calculations extends the actual scope of NGE, its functionalities and information provided, such as the microscopic simulations or the conflicts detection tool.
```

Comment:
"is that" can be replaced with a passive form: "Using both microscopic and macroscopic levels of detail brings a second benefit: […]"

"calculations" is plural, so s/extends/extend/

s/information provided/provided information/ (adjective first)